libs/blockchain/migrations/5_proxy_v2.js

```diff
@@ -0,0 +1,5 @@
+const ProxyV2 = artifacts.require("ProxyV2");
+
+module.exports = async function (deployer, network, addresses) {
+  await deployer.deploy(ProxyV2);
```

Comment:
IIUC this is the proxy that all EIP-1167 proxies point to?

If so, it's super-duper-uber critical that ownership of this proxy is "thrown out". Otherwise, an attacker could assume ownership, point this proxy to an implementation that can call `selfdestruct`, and then destroy the proxy. Now, all of the EIP-1167 proxies point to an address with no code, and everyone's funds are lost forever.

Suggest we call `initialize` right after this and point ownership to 0x1 (IIRC we can't do 0x0)

Reply:
Yes great point! We should totally thread this.